docs/serving/installing-istio.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Performing a Custom Istio Installation"
+weight: 15
+type: "docs"
+---
+
+Use this guide to perform a custom installation of Istio for use with Knative.
+
+## Download Istio
+
+Run below command to download Istio.
+```shell
+# Download and unpack Istio
+ISTIO_VERSION=1.1.3
+DOWNLOAD_URL=https://github.com/istio/istio/releases/download/${ISTIO_VERSION}/istio-${ISTIO_VERSION}-linux.tar.gz
+
+wget $DOWNLOAD_URL
+tar xzf istio-${ISTIO_VERSION}-linux.tar.gz
+
+cd istio-${ISTIO_VERSION}
+```
+
+## Install Istio CRDs 
+Default Istio Installation
+Run below command to install Istio CRDs first.
+```shell
+for i in install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl apply -f $i; done
+```
+wait a few seconds for the CRDs to be committed in the Kubernetes API-server
+
+## Custom Installation
+
+### Istio with Sidecar Injector
+If you need sidecar injection in your cluster, sidecar injector and related 
```

Comment:
How will people know if they need sidecar injection or not? Let's add some guidance here around that.

Reply:
Thanks for the suggestion! Added more context about this.